research/deeplab/deeplab_demo.ipynb

```diff
@@ -294,13 +294,13 @@
         "  try:\n",
         "    f = urllib.request.urlopen(url)\n",
         "    jpeg_str = f.read()\n",
-        "    original_im = Image.open(BytesIO(jpeg_str))\n",
+        "    orignal_im = Image.open(BytesIO(jpeg_str))\n",
```

Comment:
could you revert changes to this file? This is a recent typo fix introduced in https://github.com/tensorflow/models/pull/4518